doc/source/ray-core/user-spawn-processes.rst

```diff
@@ -0,0 +1,106 @@
+Lifetimes of a User-Spawn Process
+=================================
+
+To avoid leaking user-spawned processes, Ray provides mechanisms to kill all user-spawned processes when a worker that starts it exits. This feature prevents GPU memory leaks from child processes (e.g., torch).
+
+We have 2 environment variables to handle subprocess killing on core worker exit:
+
+- ``RAY_kill_child_processes_on_worker_exit`` (default ``true``): Only works on Linux. If true, core worker kills all subprocesses on exit. This won't work if the core worker crashed or was killed by a signal. If a process is created as a daemon, e.g. double forked, it will not be killed by this mechanism.
```

Comment:
> e.g. double forked, it will not be killed by this mechanism.

I don't think people can understand this. Either remove it or add more details?

Reply:
changed to "grandchild processes" which should be more understandable.